layouts/partials/social-links.html

```diff
@@ -1,52 +1,60 @@
 <nav class="social-links-footer" aria-label='{{i18n "contact-us"}}' >
-  <span id="contact-us-links">{{i18n "contact-us"}}</span>
 
-  <p style="display: none" aria-hidden="false">{{ i18n "options-below" }}</p>
-  <ul class="contact-info contact-info-social">
-    <li>
-      <div class="contact-icon">
-        <a title="Twitter" href="https://twitter.com/{{ i18n "twitter" }}">
-          <i class="icon-twitter"></i>
-        </a>
-      </div>
-    </li>
-    <li>
-      <div class="contact-icon">
-        <a title="LinkedIn"
-           href="https://linkedin.com/company/{{$.Site.Data.contact.linkedin}}">
-          <i class="icon-linkedin"></i>
-        </a>
-      </div>
-    </li>
-    <li>
-      <div class="contact-icon">
-        <a title="Github"
-           href="https://github.com/{{.Site.Data.contact.github}}">
-          <i class="icon-github"></i>
-        </a>
-      </div>
-    </li>
-    <li>
-      <div class="contact-icon">
-        <a title="Youtube"
-           href="https://www.youtube.com/c/CanadianDigitalServicenumériquecanadien">
-          <i class="icon-youtube"></i>
-        </a>
+  <span id="contact-us-links"></span>
+    <div class="row equal">
```

Comment:
I think this got an extra indent level from the span above which is already closed?